### PR TITLE
User profile page

### PR DESCRIPTION
**Pull request by zmustapha**
* UI change for the user profile page and all the tab contents
* Removed bio from display of profile page
* Removed photo height and width in the form for user profile

### CODE REVIEW

File: media/css/main.css [hunk 1]
```diff
@@ -382,13 +382,18 @@ footer.page-footer {
...
+.dashboard .object-action-set,
+.user-profile-page .object-action-set {
   margin: 30px 0 10px 0;
   padding: 0;
   float: none;
   background-image: none;
 }
 
+.user-profile-page .object-action-set {
+  margin-top: 0;
```
Comment (nikolas) [inline media/css/main.css]: Not a big deal here, but we tend to use 4 spaces for indentation, not 2.

**Comment by zmustapha** [comment]
I followed Google CSS styleguide:
https://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml

**Comment by nikolas** [comment]
Ah okay great. I wasn't sure what style we were following for HTML/CSS. I'll start following this too.

**Comment by zmustapha** [comment]
Let me clarify, I *try* to follow. 80% successful. 70%-ish

**Comment by sdreher** [comment]
Can we think about sticking with 4 spaces for everything? Otherwise editor setup can be a pain.

**Comment by zmustapha** [comment]
I used to have 4 spaces. Then someone recommended in the tech meeting to follow Google styleguide, so that's what I've been doing. Be happy to switch back. As long as we are all consistent, I have no preference either way.

**Comment by sdreher** [comment]
Okay - I vote a gajillion times for Google style guide but 4 spaces.

**Comment by zmustapha** [comment]
Ship it :shipit:

**Comment by thraxil** [comment]
fwiw, we also started with the google style guide on JS, but went with 4 spaces instead of 2 there, so there's precedent.

For CSS, I think we should stick with what the designers are most comfortable with, but a standard would be good.

**Pull request merged**